build/squire-raw.js

```diff
@@ -4215,7 +4215,7 @@ proto.insertImage = function ( src, attributes ) {
     return img;
 };
 
-var linkRegExp = /\b((?:(?:ht|f)tps?:\/\/|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,}\/)(?:[^\s()<>]+|\([^\s()<>]+\))+(?:\((?:[^\s()<>]+|(?:\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))|([\w\-.%+]+@(?:[\w\-]+\.)+[A-Z]{2,}\b)/i;
+var linkRegExp = /\b(?:(?:ht|f)tps?:\/\/)?(?:[\da-z\.-@]+)\.(?:[a-z\.]{2,6})(?:[\/\w\.-?#]*)*/i;
```

Comment:
Where is this regex from? I think we have a few different url regexes in the code base